sunpy/map/sources/sdo.py

```diff
@@ -130,6 +144,18 @@ def observatory(self):
         """
         return self.meta.get('telescop', '').split('/')[0]
 
+    @property
+    def reference_date(self):
```

Comment:
A idle thought, how much other stuff is duplicated between the AIA and HMI sources? Would it be worth having a `SDOMap` class with the shared stuff?

Reply:
Future PR! =)

Reply:
yes totally.